### ISSUE TITLE
Calculator seems to be broken

### ISSUE THREAD

**Issue by Ghoughpteighbteau** [getguesstimate/guesstimate-app#996]
I can't load any calculators at all.

Here's the URL for a test calculator made by me: https://www.getguesstimate.com/calculators/423

Loading it in Firefox gets me this result: https://screenshots.firefox.com/uWBO9catm1BDGwAh/www.getguesstimate.com

I can see the calculator when I edit it from the model: https://screenshots.firefox.com/0boRRFnsOqPNb6us/www.getguesstimate.com (there's clearly some kind of formatting error happening here.)

If I expand it, I can see it at that URL: https://screenshots.firefox.com/iXj6gZ3zwkfVLK7V/www.getguesstimate.com (Formatting error is still there)

**Comment by michK** [comment]
Yeah also broken for me. Such a cool tool but it's sad that development has stopped. It's also been very buggy for me before it stopped working completely.

**Comment by patbl** [comment]
When I try to load https://www.getguesstimate.com/calculators/423, I see `Uncaught TypeError: Cannot read property 'id' of null` in the browser console.

[Sentry says](https://sentry.io/organizations/guesstimate/issues/767997382/?project=60378&query=is%3Aunresolved&statsPeriod=14d&utc=true) this error started four months ago (link not publicly accessible).

The immediate cause is that `org_users` is being set to `[null]`, and then we're trying to call `null.id`:

https://github.com/getguesstimate/guesstimate-app/blob/0e7f5081fe0337060eb6a149ad2ac7e27cb910f6/src/lib/engine/space.js#L88-L90

But I don't why this is happening. There haven't been any code changes since long before this error began (according to Sentry).

@OAGr Do you have any ideas about why this is happening?

**Comment by OAGr** [comment]
Hm... it may have to do with me changing the dependencies recently, not sure. Should be able to look into it a bit this week.

**Comment by OAGr** [comment]
Yea, calculators are a bit messed up. Few people were using them earlier.

On the project, sorry to hear it stopped working completely for you @michK. I don't have many resources currently to maintain/develop it, though most of the users I know seem to have it work fine. There shouldn't be many regressions.

**Comment by michK** [comment]
No worries, I can just imagine how much time and effort it must take to maintain such a project. For now, it seems like it's working again for me! Thanks for making this project freely available; I've shown it to many friends who also love it.

**Comment by Ghoughpteighbteau** [comment]
oh, err, I think @michK is saying that guesstimate in general is working for him, the calculator still broken tho. :+1:

**Comment by michK** [comment]
Yeah you're right; sorry, didn't mean to highjack your thread (-:

**Comment by OAGr** [comment]
Ok, thanks for the update. Glad you/your friends like it @michK !
